dockerfiles/README.md

```diff
@@ -0,0 +1,16 @@
+The Dockerfile(s) for the build are in this directory
+
+* Dockerfile is the file used for most builds including localhost
+
+* Dockerfile.openshift is a custom file with the FROM entries modified to point
+to local images in the Openshift repository.
+
+If you're using Openshift you'll want to put both python and node into 
+into the openshift image stream then configure your build YAML like:
+
+``
```

Comment:
You're missing a a ` here. I couldn't get the suggestion to work.

Reply:
Thanks, fixed.